experimental/generation/generator/packages/library/test/generate.test.ts

```diff
@@ -154,6 +158,43 @@ describe('dialog:generate library', async () => {
         }
     })
 
+    it('LU and LG Unit Test', async () => {
+        for (let i = 1; i <= 23; i++) {
+            await gen.generate(`${unitTestSchemaPath}${i}.schema`, undefined, output, undefined, ['en-us'], undefined, false, false, true, feedback)
+            try {
+                console.log(`\n\nLG Testing schema ${i}`)
+                const templates = Templates.parseFile(`${output}/en-us/unittest${i}.en-us.lg`)
+                const allDiagnostics = templates.allDiagnostics
+                if (allDiagnostics) {
+                    let errors = allDiagnostics.filter((u): boolean => u.severity === DiagnosticSeverity.Error)
+                    if (errors && errors.length > 0) { throw new Error('LG format error') }
```

Comment:
>'LG format error' [](start = 71, length = 17)

Surface errors.

Reply:
I meant surface the actual errors.

---
In reply to: [513126667](https://github.com/microsoft/BotBuilder-Samples/pull/2854#discussion_r513126667) [](ancestors = 513126667)